pkg/accounting/db.go

```diff
@@ -25,7 +25,15 @@ type Raw struct {
 	CreatedAt       time.Time
 }
 
-//Rollup mirrors dbx.AccountingRollup, allowing us to use that struct without leaking dbx
+// BW mirrors dbx.StoragenodeBandwidthRollup, allowing us to use the struct without leaking dbx
+type BW struct {
```

Comment:
Since Raw is no longer atRest AND bw maybe we should get rid of the dataType field on the accounting Raw struct? Also, maybe we should consider renaming some things since raw used to include both?

Reply:
That is a good point! Let's leave it for now, and make these changes once we move the raws to the new tables.